.gitignore

```diff
@@ -8,6 +8,7 @@
 .parcel-cache
 coverage
 dist
+build
```

Comment:
a `build` folder besides a `dist` folder? Let's stick to one i think

Reply:
AFAIK `dist` is the convention for buildable libraries.

Reply:
I don't think there's a convention - eg, opentelemetry as well as sentry use `build`. I kinda like it more as it's more self explanatory, but no strong feelings

Reply:
`dist` is only left in `.gitignore` atm

Reply:
reverted back to `dist`